packages/web/src/services/audius-backend/WithdrawUSDC.ts

```diff
@@ -0,0 +1,224 @@
+  const quoteRoute = await JupiterSingleton.getQuote({
+    inputTokenSymbol: 'USDC',
+    outputTokenSymbol: 'SOL',
+    inputAmount: amount,
+    slippage: USDC_SLIPPAGE,
+    swapMode: SwapMode.ExactOut,
+    onlyDirectRoutes: true
+  })
+  const usdcNeededAmount = quoteRoute.inputAmount
+
+  const swapRoute = await JupiterSingleton.getQuote({
+    inputTokenSymbol: 'USDC',
+    outputTokenSymbol: 'SOL',
+    inputAmount: usdcNeededAmount.uiAmount,
+    slippage: USDC_SLIPPAGE,
+    onlyDirectRoutes: true
+  })
+  const exchangeInfo = await JupiterSingleton.exchange({
+    routeInfo: swapRoute.route,
+    userPublicKey: solanaRootAccountPubkey,
+    feeAccount: feePayerPubkey
+  })
+  const swapInstructions =
+    exchangeInfo.transactions.swapTransaction.instructions
+
+  const transferInstructions =
+    await libs().solanaWeb3Manager!.createTransferInstructionsFromCurrentUser({
+      amount: usdcNeededAmount.uiAmount,
+      feePayerKey: feePayerPubkey,
+      senderSolanaAddress: usdcUserBank,
+      recipientSolanaAddress: solanaUSDCATA,
+      instructionIndex: 1,
+      claimableTokenPDA: libs().solanaWeb3Manager!.claimableTokenPDAs.usdc,
+      solanaTokenProgramKey: TOKEN_PROGRAM_ID,
+      claimableTokenProgramKey:
+        libs().solanaWeb3Manager!.claimableTokenProgramKey,
+      connection,
+      mint: 'usdc'
+    })
+
+  const createATAInstruction = await createUSDCATAInstruction(
+    solanaUSDCATA,
+    solanaRootAccountPubkey,
+    feePayerPubkey
+  )
+  const closeATAInstruction = await closeUSDCATAInstruction(
+    solanaUSDCATA,
+    solanaRootAccountPubkey,
+    feePayerPubkey
+  )
+
+  const instructions = [
+    createATAInstruction,
+    ...transferInstructions,
+    ...swapInstructions,
+    closeATAInstruction
+  ]
+  return await libs().solanaWeb3Manager!.transactionHandler.handleTransaction({
+    instructions,
+    skipPreflight: true,
+    feePayerOverride: feePayerPubkey
+  })
+}
+
+const getUSDCATA = async (solanaRootAccountPubkey: PublicKey) => {
+  return await Token.getAssociatedTokenAddress(
+    ASSOCIATED_TOKEN_PROGRAM_ID,
+    TOKEN_PROGRAM_ID,
+    libs().solanaWeb3Manager!.mints.usdc,
+    solanaRootAccountPubkey
+  )
+}
+
+const createAndFundDestinationATA = async (
+  destinationPubkey: PublicKey,
+  feePayerPubkey: PublicKey
+) => {
+  // TODO: factor in existing sol balance
+  // TODO: might have to pay rent for root sol account, see BuyAudio.ts
+  const desiredSolAmount =
+    (await getRentAndTransferTransactionFee(destinationPubkey)) /
+    LAMPORTS_PER_SOL
+
+  const solanaRootAccount = await getRootSolanaAccount()
+  const usdcUserBank = await libs().solanaWeb3Manager!.deriveUserBank({
+    mint: 'usdc'
+  })
+  const solanaUSDCATA = await getUSDCATA(solanaRootAccount.publicKey)
+  swapUSDCForSol(
+    desiredSolAmount,
+    solanaRootAccount.publicKey,
+    usdcUserBank,
+    solanaUSDCATA,
+    feePayerPubkey
+  )
+}
+
+export const getOrCreateDestinationATA = async (
```

Comment:
Do you mind adding jsdocs to exported functions so it's easier to understand them when using in calling code?